qa-portal-angular/projects/qa-forms/src/app/controls/radio-button/radio-button.component.scss

```diff
@@ -2,50 +2,37 @@
   display: flex;
   flex-direction: column;
   font-size: 14px;
+  margin-top: 8px;
 }
 
 .qa-radio-btn-grp-label {
-  background-color: lightgrey;
-  color: black;
+  background-color: #004050;
```

Comment:
Consider using the material theme to get this colour instead of hard-coding it.

Reply:
I am still a little unsure on how to set up the material theme for the checkbox and radio button components. Do I need to create a general scss file in qa-forms > environments as with the qa-app-theme.scss file in portal-side-menu > environments?